types/wlr_data_device.c

```diff
@@ -815,3 +815,11 @@ struct wlr_data_device_manager *wlr_data_device_manager_create(
 
 	return manager;
 }
+
+void wlr_data_device_manager_destroy(
+    struct wlr_data_device_manager *manager) {
```

Comment:
These would fit on one line?

Reply:
Ah whoops, renamed it and didn't realize it went back under 80 columns